docs/tempo/website/configuration/_index.md

```diff
@@ -123,13 +123,16 @@ storage:
             bucket_name: ops-tools-tracing-ops   # store traces in this bucket
         blocklist_poll: 5m                       # how often to repoll the backend for new blocks
         blocklist_poll_concurrency: 50           # optional. Number of blocks to process in parallel during polling. Default is 50.
-        cache: memcached                         # optional cache configuration
-        memcached:                               # optional memcached configuration
+        cache: memcached                         # optional. Cache configuration
+        background_cache:                        # optional. Background cache configuration
```

Comment:
I think it would be valuable to mention somewhere that this needs to be used together with a "real" cache, because it can be non-obvious on first look

Reply:
Added a small note about this. Looks better?